scripts/yast2

```diff
@@ -13,6 +13,14 @@
 # wise ncurses. It starts then the module 'menu' which implements
 # the configuration and administration menu.
 
+# FATE#317637, bsc#877447: In installation system, we call the installer
+# program directly
+if [ -f /.instsys.config ]; then
+  echo "Starting Installer"
+  exec /usr/lib/YaST2/startup/YaST2.First-Stage
+  exit $?
```

Comment:
BTW this `exit` will only be run if the `exec ` failed, but that is actually what we want :)